oidc-api/src/main/java/uk/gov/di/authentication/oidc/helpers/RequestObjectToAuthRequestHelper.java

```diff
@@ -41,7 +42,19 @@ public static AuthenticationRequest transform(AuthenticationRequest authRequest)
                             .requestObject(authRequest.getRequestObject());
 
             if (Objects.nonNull(jwtClaimsSet.getClaim("vtr"))) {
```

Comment:
I think that this could be refactored to remove a layer of nesting:

```
if (vtrClaim instanceof String) { ... }
else if (vtrClaim instanceof List && all elements strings) { ... }
else if (vtrClaim nonNull) { return error}
else { // set to empty list }
```

Reply:
We don't actually do anything if this is null though? We just skip all these steps

Reply:
Oh okay actually I think I misunderstood a bit - we just don't need the else in your block above, because nothing happens in that case. I can see what you mean, but I'm still not entirely convinced - I think the logic reads a bit more clearly this way - in that if it's null, we don't need to anything, then if it's not null, we act based off of type, rather than combining the logic